worker/main.ts

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env -S deno run --allow-read=. --allow-net --allow-env
 /* Copyright 2020 the Deno authors. All rights reserved. MIT license. */
 
-import { handleRequest, withLog } from "./handler.ts";
+import { handleRequest } from "./handler.ts";
 import { listenAndServe } from "https://deno.land/std@0.108.0/http/server.ts";
 
-const handler = withLog(handleRequest);
-
 console.log("The server is available at http://localhost:8081");
-listenAndServe(":8081", handler);
+listenAndServe(":8081", handleRequest);
```

Comment:
`//worker/main.ts` should just be a symlink to `//main.ts` ?

Reply:
yes, this is just old code to keep reference with while porting